.env.dev

```diff
@@ -1,4 +1,7 @@
-PORT=3000
+PORT=8000
 NODE_ENV=development
 JWT_SECRET=thisismysupersecrettokenjustkidding
-DATABASE_URL=mongodb://localhost:27017/donut-development
+DATABASE_URL=mongodb+srv://asel123:asel123@socialmediaappcluster-leoir.mongodb.net/test?retryWrites=true&w=majority
+GOOGLE_CLIENT_ID=880828187640-v97ss4hngceeermqc3ca3sbq3hlc7f1c.apps.googleusercontent.com
+GOOGLE_CLIENT_SECRET=sUjZ0ruvsdo7GADb6C0tLjr3
```

Comment:
this too

Reply:
@devesh-verma Thanks for pointing out. My bad. I was under the impression that the .env files were in the .gitignore. @vaibhavdaren Also pointed out some committed keys in the PR on confirmation email. 

I'm currently at my grandparent's place working on my proposal and I don't have the access to my university laptop at the moment. I will be returning the day after tomorrow. Could these changes wait until I return?

Reply:
Yeah sure, just let me know once done.